GooglePlayInstant/Editor/QuickDeploy/LoadingScreenGenerator.cs

```diff
@@ -131,9 +141,15 @@ internal static void AddScript(GameObject loadingScreenGameObject)
         internal static void AddImageToScene(GameObject loadingScreenGameObject,
             string pathToLoadingScreenImage)
         {
-            loadingScreenGameObject.AddComponent<Canvas>();
```

Comment:
Is there any harm in doing the same action every time like we did before? The code is a bit more complex now.

Reply:
Yeah. It will throw an error if multiple canvas/image objects are attempted to be put onto the same object. To avoid creating a function that would reuse mostly the same code, I refactored it so it would not throw an error if used multiple times on the same scene.